### PR TITLE
Added full integration test suite.

### PR DESCRIPTION
**Pull request by n1k0**
This patch adds integration tests by setting up a local kinto server instance and run a test suite against it.

r=? @ametaireau @Natim @leplatrem

### CODE REVIEW

**Comment by n1k0** [comment]
This is now ready for review. Take your time. r=? @ametaireau @Natim @leplatrem

**Comment by ametaireau** [comment]
Very cool to see how the actual code is impacted. Proves @Natim point about unit tests :x

**Comment by ametaireau** [comment]
r+

File: src/collection.js [hunk 1]
```diff
@@ -169,14 +169,14 @@ export default class Collection {
    * @param  {Object} options
    * @return {Promise}
    */
-  create(record, options={synced: false}) {
+  create(record, options={forceUUID: false, synced: false}) {
```
Comment (ametaireau) [inline src/collection.js]: Should we expose this in the documentation somehow?

Comment (n1k0) [inline src/collection.js]: I debated this, but I couldn't see any use case for end users to use this option, which I've added for test context creation only. Your call.

**Pull request merged**